### PR TITLE
fix(kms): reduce severity for possible `scylla-bench` failures in KMS nemesis

### PR DESCRIPTION
**Pull request by vponomaryov**
Running the KMS nemesis we may get following error in one of the scylla-bench stress commands:

```
  Stress command completed with bad status 1: 2023/10/03 16:01:04 \
    Error during truncate: seastar::rpc::remote_verb_error (filesystem error: stat f
```

And following one in DB logs:

```
  sstable - Error while linking SSTable: /var/.../tmp_encrypted_table.*.db \
   to /var/.../tmp_encrypted_table-.*/snapshots/.*.db: \
   std::filesystem::__cxx11::filesystem_error (error system:2, \
   filesystem error: stat failed: No such file or directory \
   [/var/.../tmp_encrypted_table-.*.db])
```

It started happening after addition of the `-validate-data` flag to the `write` stress command which does `truncate` in this case.

The addition of the `-validate-data` flag is required to make the data validation also work in the `read` stress load.

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

**Comment by vponomaryov** [comment]
Example of the error events which are marked here as `warning`:
https://argus.scylladb.com/test/631c0991-5548-434e-b7a3-f47eb2db8777/runs?additionalRuns[]=245363be-fa67-4779-b1c2-142c42f63abf

Testing the PR here: https://argus.scylladb.com/test/631c0991-5548-434e-b7a3-f47eb2db8777/runs?additionalRuns[]=6d42d8cf-11dc-4100-b5f3-042680719556

File: sdcm/nemesis.py [hunk 1]
```diff
@@ -4012,15 +4012,30 @@ def check_encryption_fact(sstable_util_instance, expected_bool_value):
             assert encryption_results
             assert all(encryption_result is expected_bool_value for encryption_result in encryption_results)
 
+        def run_write_scylla_bench_load(write_cmd):
+            # NOTE: 'scylla-bench' runs 'truncate' operation when 'validate-data' is used in addition
```
Comment (fruch) [inline sdcm/nemesis.py:4016]: it's that a issue/bug to fix on s-b ? 
or I don't understand why are we o.k. with it failing ?

Comment (vponomaryov) [reply sdcm/nemesis.py:4016]: It is concurrency.
The full error from the scylla-bench looks like this:

```
2023/10/03 15:53:49 Error during truncate: seastar::rpc::remote_verb_error (filesystem error: stat failed: No such file or directory [/var/lib/scylla/data/keyspace1/tmp_encrypted_table-c8b6abb0620411eeb3bbffe61f372cec/me-3g9w_184y_38t8w2qwwiu866t6n7-big-Scylla.db])
```

The same on the DB level:

```
2023-10-03T15:53:49+00:00 longevity-10gb-3h-dev-db-node-245363be-1      !ERR | scylla[14449]:  [shard 0:sl:d] sstable - Error while linking SSTable: /var/lib/scylla/data/keyspace1/tmp_encrypted_table-c8b6abb0620411eeb3bbffe61f372cec/me-3g9w_184y_38t8w2qwwiu866t6n7-big-Scylla.db to /var/lib/scylla/data/keyspace1/tmp_encrypted_table-c8b6abb0620411eeb3bbffe61f372cec/snapshots/1696348429426-tmp_encrypted_table/me-3g9w_184y_38t8w2qwwiu866t6n7-big-Scylla.db: std::filesystem::__cxx11::filesystem_error (error system:2, filesystem error: stat failed: No such file or directory [/var/lib/scylla/data/keyspace1/tmp_encrypted_table-c8b6abb0620411eeb3bbffe61f372cec/me-3g9w_184y_38t8w2qwwiu866t6n7-big-Scylla.db])
```

Sstables concurrently get removed. It is expected behavior.

Comment (vponomaryov) [reply sdcm/nemesis.py:4016]: Also, our goal is to check that `existing` sstables are `encrypted`. It gets checked by further assertions.

Comment (fruch) [reply sdcm/nemesis.py:4016]: this whole using of `--validata-data` on write load, isn't clear to me.
also letting s-b do truncate, sound a bit off to me... don't something I would expected from a stress tool like that.

but for now, I'll leave it as is

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**